frontend/src/components/common/NamespacesAutocomplete.tsx

```diff
@@ -153,7 +155,31 @@ export function PureNamespacesAutocomplete({
             {tags.length > joinnedNamespaces && (
               <>
                 <span>,&nbsp;</span>
-                <b>{`+${tags.length - joinnedNamespaces}`}</b>
+                <b
+                  style={{
+                    cursor: 'pointer',
+                  }}
+                  onMouseEnter={() => setIsHovered(true)}
+                  onMouseLeave={() => setIsHovered(false)}
+                >{`+${tags.length - joinnedNamespaces}`}</b>
+
+                {isHovered && (
+                  <ul
```

Comment:
For a more consistent UI I'd suggest using [Tooltip component](https://v5.mui.com/material-ui/react-tooltip/) that we also use in other places